app/src/main/java/com/spacelobster/fuckingbills/fragment/HelloFragment.kt

```diff
@@ -88,7 +82,8 @@ class HelloFragment : Fragment(), AnkoLogger {
         binding.house.setOnDragListener(onHouseDragListener)
 
         binding.nextBtn.setOnClickListener {
-            storeCounters()
+            callback!!.onCountersSelected()
+            Completable.fromAction { storeCounters() }
```

Comment:
not sure what is better to use here. Put completable just because why not, but we need to fire callback!!.onCountersSelected() after all the counters being stored

Reply:
The room transactions won't work in UI thread, so I gonna use second approach with subscribe from the place where we need the data. Yeew